src/prefix.js

```diff
@@ -0,0 +1,27 @@
+/* @flow */
+
+import canUseDOM from 'can-use-dom';
+
+let prefix;
+
+if (canUseDOM) {
+    // Client-side, we can use the dynamic prefixer to avoid unnecessary work.
+    const Prefixer = require('inline-style-prefixer');
+
+    // We need keepUnprefixed so our sorting code knows how to order the
+    // prefixed styles.
+    const prefixer = new Prefixer({
```

Comment:
Aside from server-side rendering, I know one of the reasons we initially didn't go with this approach is because the code to determine which prefixes were needed (which we're doing a build time in `staticPrefixData`) took up a lot of space. Can you build aphrodite before and after this change and see how much bigger the resulting dist file is?